docs/cep/source/source-types/stream-source.md

```diff
@@ -11,24 +11,50 @@ The stream processor groups common types of events together with a schema. This
 By default, all streams created by stream workers are sources. The syntax for defining a new stream in a stream worker is:
 
 ```sql
-CREATE STREAM <stream_name> (<attribute_name> <attribute_type>,
+   CREATE SOURCE SinkName WITH (
+      type="stream", 
+      stream.list="STRING[STRING,]", 
+      replication.type="STRING", 
+      subscription.initial.position="STRING",
+      num.consumers="INT",
+      num.listener.threads="INT",
+      num.io.threads="INT",
+      receiver.queue.size="INT",      
+      map.type='type')  
+    (<attribute_name> <attribute_type>[,
+     <attribute_name> <attribute_type>] );
+```
+
+Or you can use the syntax shortcut `CREATE STREAM`:
+
+```sql
+CREATE STREAM <stream_name> <GLOBAL | LOCAL> (<attribute_name> <attribute_type>,
                              <attribute_name> <attribute_type>, ... );
 ```
+Or you can use the syntax shortcut for local stream `CREATE STREAM`:
+
+```sql
+CREATE SINK STREAM  SampleStreamSink (data string);
+```
+
 
 ## Parameters
 
 The following parameters are used to configure a stream definition.
 
-| Parameter     | Description |
-| ------------- |-------------|
-| `stream name`      | The name of the stream created. (By convention, stream names use `PascalCase`.) |
-| `attribute name`   | Uniquely identifiable name of the stream attribute. (By convention, attribute names are defined in `camelCase`.)|     |
-| `attribute type`   | The type of each attribute defined in the schema. This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
+| Parameter     | Description | Default Value | Possible Data Types | Optional |
+| ------------- |-------------| ------------- | ------------------- | -------- |
+| stream.list | The list of streams the sources will consume events.| -    | STRING        | No                  |
+| replication.type | Specifies if the replication type of the streams. Possible values can be `LOCAL` and `GLOBAL`.      | LOCAL         | STRING         | Yes      |
+| subscription.initial.position | Subscription's initial position in the stream. Possible values: [Earliest,Latest]| Latest | STRING | Yes|
+| `subscription.type` | Stream subscription type. Possible Values: [Exclusive, Shared, Failover, Key_Shared]. | Shared | STRING | Yes|
+| `num.consumers` | Number of consumers | 1 | INT | Yes |
+| `num.listener.threads` | the number of listener threads | 1 | INT | Yes |
```

Comment:
```suggestion
| num.listener.threads | The number of listener threads. | 1 | INT | Yes |
```

Reply:
What do you mean by "threads" here? I have not seen "threads" used in our docs thus far. Do you mean streams? Subscriptions?

Reply:
@diana-macrometa - The `threads` either `listener` or `io` are part of pulsar consumer spec. 
This is similar to the `threading.option` in `Kafka-source`.